### PR TITLE
Complete nuke gulp + redundant russian file

### PR DESCRIPTION
**Pull request by ProstoSanja**
Remove all gulp mentions
there are two files with exactly same content in russian, i deleted one with english title and remainde one with russian

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "angel_arena_allstars",
-  "version": "0.0.0",
+  "name": "open_angel_arena",
+  "version": "0.0.1",
```
Comment (chrisinajar) [inline package.json]: we're not at 0.1 yet, and we'd use npm cli to do it anyway (for auto-tagging and all that fanciness)

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   "description": "A really great Dota 2 game mode.",
-  "main": "index.js",
+ // "main": "index.js",
```
Comment (chrisinajar) [inline package.json]: .json doesn't support comments, this will throw errors later

File: package.json (hunk 1 above)
Comment (ProstoSanja) [inline package.json]: do you want me to revert version AND name?

Comment (Chronophylos) [inline package.json]: Version only

**Comment by ProstoSanja** [comment]
Ready for merge

**Comment by ProstoSanja** [comment]
@chrisinajar

**Pull request merged**